File: marktia-frontend/dtos/responses/federations/federationResponseDTO.ts
import { DistrictResponseDTO } from "./countyResponseDTO";
import { RegionResponseDTO } from "./regionResponseDTO";
import { StateResponseDTO } from "./stateResponseDTO"

export type FederationResponseDTO = {
    state: StateResponseDTO;
    region: RegionResponseDTO;
    county: DistrictResponseDTO;
}

File: marktia-backend/src/main/java/ufes/marktiabackend/dtos/responses/federation/FederationResponseDTO.java
package ufes.marktiabackend.dtos.responses.federation;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class FederationResponseDTO {
    private FederationFieldResponseDTO state;

    private FederationFieldResponseDTO region;

    private FederationFieldResponseDTO county;
}
